2.train-VAE/scripts/1.optimize_hyperparameters.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import sys
+import pathlib
+import numpy as np
+import pandas as pd
+import hiplot
+from optuna.visualization import plot_param_importances
+import optuna
+import torch
+from torch.utils.data import DataLoader, TensorDataset
+from sklearn.model_selection import train_test_split
+
+from betavae import (
+    BetaVAE,
+    train_vae,
+    evaluate_vae
+)
+from optimize_utils import(
+    get_optimize_args
+)
+
+
+# In[ ]:
+
+
+#Load command line arguments
+args=get_optimize_args()
+
+#Load data
+
+output_dir = pathlib.Path("data")
+
+sys.path.insert(0, "../0.data-download/scripts/")
+from data_loader import load_train_test_data
+data_directory = pathlib.Path("../0.data-download/data")
+dfs = load_train_test_data(data_directory, train_or_test = "all", load_gene_stats = True)
+
+train_feat = dfs[0]
+test_feat = dfs[1]
+load_gene_stats = dfs[2]
```

Comment:
I remember this code!

Consider loading all three immediately, rather than having code saved to redundant variables:

```python
train_feat, test_feat, load_gene_stats = load_train_test_data(...)
```

Also, I think these variable names can be improved :) What is a "feat"? 🦶 😂